docs(guides): Add "npm build" instruction on "Loading Data" section

Vanary opened the pull request:
Added instruction for `npm build` on the `guides/asset-management`.

This change would prevent reader from confusion if they forgot to build module before check upon `dist/index.html`.

EugeneHlushko commented on src/content/guides/asset-management.md:
````diff
@@ -483,6 +483,18 @@ __src/index.js__
   document.body.appendChild(component());
 ```
 
+Now once again, run your build command:
````
We had already given example output of the build command and instruction to run the build itself, i would suggest adding this info to the next paragraph e.g.
```
Re-run the build command and open `index.html`....
```

Vanary commented:
I like the simplicity of suggestion. Followed review by:
- Removed build & CLI result from first commit
- Added information to re-run build command as suggested

EugeneHlushko commented:
Thanks!
